hnn_core/network.py

```diff
@@ -76,27 +76,27 @@ def __init__(self, params, n_jobs=1):
         # this is a deviation from the original, where L5 was defined at 0
         # this should not change interlaminar weight/delay calculations
         self.zdiff = 1307.4
-        # params of external inputs in p_ext
+        # params of common external feeds inputs in p_common
         # Global number of external inputs ... automatic counting
         # makes more sense
         # p_unique represent ext inputs that are going to go to each cell
-        self.p_ext, self.p_unique = create_pext(self.params,
-                                                self.params['tstop'])
-        self.N_extinput = len(self.p_ext)
+        self.p_common, self.p_unique = create_pext(self.params,
+                                                   self.params['tstop'])
+        self.N_common_feeds = len(self.p_common)
```

Comment:
```suggestion
        self.n_common_feeds = len(self.p_common)
```
like in MNE and other packages. All small letters.